tests/benchdnn/utils/cfg.hpp

```diff
@@ -185,7 +185,7 @@ struct base_cfg_t {
         }
         const int64_t safe_digits = get_safe_digits();
         const int64_t safe_n_acc = (1LL << safe_digits) / max_value;
-        return safe_n_acc;
+        return std::max((int64_t)1L, safe_n_acc);
```

Comment:
Returning `safe_n_acc = 0` is intentional here, it says that input values are not reasonable. When it happened it returned zero?

Reply:
removed, as there dont appear to be affected cases with latest changes.